components/segmenter/tests/testdata/LineBreakRandomTest.txt

```diff
@@ -0,0 +1,205 @@
+# The following 200 monkey tests were generated from
+# https://github.com/makotokato/icu/tree/monkey-tests-untailor-lb-74-2
+# (specifically, at f9e2c40abd917274cbc8db4b57b6fd5a438b23dc)
+# by running
+#   intltest.exe "rbbi/RBBITest/TestMonkey@type=line seed=1729 loop=1000 scalars_only export=meow"
```

Comment:
LineBreakExtraTest had the comment
```
# When actually working on changes to the rules, much longer runs should be used.
```
Please retain this comment, and mention in the PR description the number of tests you actually used when testing the change.